samples/subsys/fs/fat_fs/README.rst

```diff
@@ -23,7 +23,7 @@ both the ``nrf52840_blip.overlay`` and the ``dts_fixup.h`` for nrf52840_blip
 to work:
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/subsys/fs
+   :zephyr-app: samples/subsys/fs/fat_fs
```

Comment:
Since you pushed the README.rst down one folder level, Sphinx didn't find it in the toctree wildcard  found in ``samples/subsys/subsys.rst``.  So, to be consistent you'll need to also add a ``samples/subsys/fs/fs.rst`` that mirrors other subsys doc trees with an extra level (see ``logging/logging.rst`` for example).   That will fix the docbuild error.

Reply:
Thanks; that's and passed.